gui/mozregui/build_runner.py

```diff
@@ -172,6 +173,11 @@ def start(self, fetch_config, options):
         self.thread.start()
         # this will be called in the worker thread.
         QTimer.singleShot(0, action)
+        # an action = instance of mozregression usage, so send
+        # a usage ping (if telemetry is enabled)
+        if get_prefs()['enable_telemetry']:
```

Comment:
No need to check this value here -- you can let Glean's enable / disable handle this.